Source/interface_example.py

```diff
@@ -0,0 +1,63 @@
+#
+# License: See LICENSE file
+# GitHub: https://github.com/Baekalfen/PyBoy
+#
+
+from pprint import pprint
+from pyboy import PyBoy
+
+# pyboy = PyBoy("ROMs/POKEMON BLUE.gb", window_type="headless", window_scale=1) # For unattended use, for example machine learning
+pyboy = PyBoy("ROMs/POKEMON BLUE.gb", window_type="SDL2", window_scale=1)
+pyboy.set_emulation_speed(False, 0)
+
+for n in range(1000): # Move ahead the desired number of frames.
+    pyboy.tick()
+
+tile_map = pyboy.get_window_tile_map() # Get the TileView object for the window.
+
+# The following prints out the indices of each tile in the window -- excluding sprites and the background!
+print(tile_map)
+#      0   1   2   3   4   5   6   7   8   9   10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31
```

Comment:
```suggestion
# Tile Map Address: 0x9c00, High Tile Data: No     0   1   2   3   4   5   6   7   8   9   10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  2
```

Reply:
It should actually have been on its own line. I'll make a correction.